app/proprietary/src/main/java/stirling/software/proprietary/security/configuration/SecurityConfiguration.java

```diff
@@ -119,8 +127,70 @@ public PasswordEncoder passwordEncoder() {
         return new BCryptPasswordEncoder();
     }
 
+    @Bean
```

Comment:
seems odd to bean this if you directly add it to filterchain yourself without autowired.. 

If you did autowire it the null wouldnt be allowed anyway